.dockerignore

```diff
@@ -1,6 +1,11 @@
+.git
 *.pyc
 .idea
-.git
+.coverage
 .tox
+.cache
+dist/
+build/
 docs/
```

Comment:
The two files are now identical except for `.git` in `.dockerignore`. It's probably the best this way.